lab_1/main.py

```diff
@@ -3,17 +3,33 @@
 A concordance extraction
 """
 
-
-def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    """
-    pass
-
+text = open('data.txt', encoding='utf-8').read()
```

Comment:
remove lines 6,7, 31, 32
1) you should do this in start.py
2) a function cannot be into if/else, it should exist on its own

**the logic of main.py**
def ...():
    ...

def ...():
    ...

def ...():
    ...

So, there should be only functions and their implementations